x-pack/plugins/security_solution/public/common/lib/cell_actions/default_cell_actions.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState, useMemo } from 'react';
+
+import type {
+  BrowserFields,
+  TimelineNonEcsData,
+} from '../../../../../timelines/common/search_strategy';
+import { DataProvider, TGridCellAction } from '../../../../../timelines/common/types';
+import { TimelineId } from '../../../../common';
+import { getMappedNonEcsValue } from '../../../timelines/components/timeline/body/data_driven_columns';
+import { IS_OPERATOR } from '../../../timelines/components/timeline/data_providers/data_provider';
+import { allowTopN, escapeDataProviderId } from '../../components/drag_and_drop/helpers';
+import { ShowTopNButton } from '../../components/hover_actions/actions/show_top_n';
+import { getAllFieldsByName } from '../../containers/source';
+import { useKibana } from '../kibana';
+
+/** a noop required by the filter in / out buttons */
+const onFilterAdded = () => {};
```

Comment:
Noting here too: We can just make this optional in the component. But that can be done in a follow up PR